packages/oui-calendar/src/calendar.controller.js

```diff
@@ -100,6 +100,28 @@ export default class {
         this.initCalendarInstance();
     }
 
+    $onChanges ({ minDate, maxDate }) {
+        console.log(this.flatpickr);
```

Comment:
please remove console.log

Reply:
Nice catch. That's the bad side of willing doing this fast :(